doc/api/tls.markdown

```diff
@@ -814,6 +815,7 @@ The numeric representation of the local port.
 [Chrome's 'modern cryptography' setting]: http://www.chromium.org/Home/chromium-security/education/tls#TOC-Deprecation-of-TLS-Features-Algorithms-in-Chrome
 [specific attacks affecting larger AES key sizes]: https://www.schneier.com/blog/archives/2009/07/another_new_aes.html
 [BEAST attacks]: http://blog.ivanristic.com/2011/10/mitigating-the-beast-attack-on-tls.html
+[crypto.getCurves()]: #crypto_crypto_getcurves
```

Comment:
This link won't work because as-is since `getCurves()` is in a different document.